src/main/kotlin/org/codefreak/codefreak/graphql/api/AnswerApi.kt

```diff
@@ -52,8 +67,11 @@ class AnswerDto(@GraphQLIgnore val entity: Answer, ctx: ResolverContext) : BaseD
   }
 }
 
+@GraphQLName("CommitEntry")
+class CommitDto(val key: Int, val value: String)
```

Comment:
I think key and value aren't very descriptive here. What key is mapped to what value?